core/spend.go

```diff
@@ -37,6 +37,7 @@ type SpendResponse struct {
 	Timestamp          time.Time `json:"timestamp"`
 	Txid               string    `json:"txid"`
 	UnconfirmedBalance int64     `json:"unconfirmedBalance"`
+	PeerID             string    `json:"-"`
```

Comment:
I don't see this used anywhere in the PR. What's the purpose of this?

Reply:
The peerID is used to create the ORDER_PAYMENT message when the spend returns.